src/utils/generalVariables.js

```diff
@@ -0,0 +1,17 @@
+const URL = "http://demo0656939.mockable.io"
+
```

Comment:
I didn't know about Mockable 😮 [What is this wizardry?!](https://i2.kym-cdn.com/photos/images/newsfeed/000/117/814/are-you-wizard.jpg) Super awesome how you did it to unblock this in a way that will be changed super fast when it's finally ready 🔥

Reply:
Mockable is awesome! It takes a fair bit of work to get it up and running though. However, it's a good prototyping tool, highly recommended!